privacyidea/translations/de/LC_MESSAGES/messages.po

```diff
@@ -862,14 +886,42 @@ msgstr "Der Text der SMS."
 msgid "Your LDAP config seems to be OK, %i machine objects found."
 msgstr "Die LDAP-Konfiguration scheint in Ordnung zu sein. Es wurden %i Maschinen-Objekte gefunden."
 
-#: lib/resolvers/LDAPIdResolver.py:959
+#: lib/resolvers/LDAPIdResolver.py:968
 msgid "Your LDAP config found {0!s} user objects, but only {1!s} with the specified uidtype"
 msgstr "Für die LDAP-Konfiguration wurden {0!s} Benutzerobjekte gefunden, aber lediglich {1!s} mit dem angegebenen UID Typ."
 
-#: lib/resolvers/LDAPIdResolver.py:962
+#: lib/resolvers/LDAPIdResolver.py:971
 msgid "Your LDAP config seems to be OK, {0!s} user objects found."
 msgstr "Die LDAP-Konfiguration scheint in Ordnung zu sein. Es wurden {0!s} Benutzerobjekte gefunden."
 
+#: lib/smsprovider/FirebaseProvider.py:114
+msgid "The URL the Push App should contact in the second enrollment step. Usually it is the endpoint /ttype/push of the privacyIDEA server."
+msgstr "Die URL für den zweiten Ausrollschritt. Diese URL ruft das Smartphone auf, nachdem der Benutzer den QR-Code gescannt hat. Überlicher Weise ist das der Endpunkt /ttype/push des privacyIDEA Servers."
+
+#: lib/smsprovider/FirebaseProvider.py:118
+msgid "How long should the second step of the enrollment be accepted (in minutes)."
+msgstr "Die Zeit, für die die Antwort im zweiten Schritt beim Ausrollen akzeptiert wird (in Minuten)."
```

Comment:
Die Zeit, in der die Anwort im zweiten Schritt des Ausrollens (noch) akzeptiert wird (in Minuten).

Reply:
done.